spec/helpers.rb

```diff
@@ -36,7 +36,7 @@ def create_ready_vm(template, name, token = nil)
   create_vm(name, token)
   redis.sadd("vmpooler__ready__#{template}", name)
   # REMIND: should be __vm__?
-  redis.hset("vmpooler_vm_#{name}", "template", template)
+  redis.hset("vmpooler__vm__#{name}", "template", template)
```

Comment:
Outside the scope of this PR, but my hope has been that we eventually refactor enough of the core parts of the pooler into methods (or ideally classes with methods) that hide the redis-y bits, and then these spec helpers just disappear in favor of using the real domain model.